models/qualtrics__contact.sql

```diff
@@ -68,7 +68,7 @@ agg_distribution_responses as (
     group by 1,2
 ),
 
--- todo: add medians
+-- todo: add medians 
```

Comment:
Were you still planning to include medians in this model?

Reply:
added!